data/src/main/java/org/apache/iceberg/data/DeleteFilter.java

```diff
@@ -54,35 +56,42 @@
 import org.apache.iceberg.util.Filter;
 import org.apache.iceberg.util.StructLikeSet;
 import org.apache.iceberg.util.StructProjection;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public abstract class DeleteFilter<T> {
-  private static final long DEFAULT_SET_FILTER_THRESHOLD = 100_000L;
+  private static final Logger LOG = LoggerFactory.getLogger(DeleteFilter.class);
+  public static final long DEFAULT_STREAM_FILTER_THRESHOLD = 100_000L;
```

Comment:
Can we extract all of these set/streamFilter changes out into another PR?

Reply:
I will revert the last commit from this PR. If this PR can be merged without that, I'll open a separate PR once it is merged, consisting of that change.